app/constants/constants.js

```diff
@@ -178,7 +180,21 @@ export const CONNECTION_CONFIG = {
             'value': 'secretAccessKey',
             'type': 'password'
         }
-    ] // TODO - password options for apache drill
+    ], // TODO - password options for apache drill
+    [DIALECTS.DATA_WORLD]: [
+        {
+            'label': 'Dataset/Project URL',
+            'value': 'url',
+            'type': 'text',
+            'description': 'The URL of the dataset or project on data.world'
+        },
+        {
+            'label': 'Token',
```

Comment:
`data.world` provides two API tokens: `Read/Write` and `Admin`.

To help users, we could re-label this input as `Read/Write API Token`.